tao_compiler/mlir/disc/tests/regression/empty_tensor.cc

```diff
@@ -37,7 +38,8 @@ TEST(EmptyTensor, FullyDynamicShape2DF32) {
 TEST(EmptyTensor, ImplicitBroadcast2DF32) {
   EXPECT_TRUE(feature_test_main(
```

Comment:
maybe we can add a function `feature_test_main_with_all_backend`, so that we don't need to update all `.cc` files on each new backend?

Reply:
Thanks. I define some helper backend lists instead of explicitly listing the backends.